src/adr-new

```diff
@@ -76,7 +76,7 @@ fi
 newid=$(printf "%04d" $newnum)
 slug=$(echo -n $title | tr -Ccs [:alnum:] - | tr [:upper:] [:lower:] | sed -e 's/[^[:alnum:]]*$//' -e 's/^[^[:alnum:]]*//')
 dstfile=$dstdir/$newid-$slug.md
-date=${ADR_DATE:-$(date +%d/%m/%Y)}
+date=${ADR_DATE:-$(date +%F)}
```

Comment:
The date command on MacOS X doesn't support %F.  I'll manually merge the ADR for this change, and implement the date format explicitly so that it works on MacOS X  and Linux.